scripts/functions/helper_ae.py

```diff
@@ -0,0 +1,128 @@
+#----------------------------------------------------------------------------------------------------------------
+# Helper functions for running autoencoder
+#----------------------------------------------------------------------------------------------------------------
+
+import os
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import tensorflow as tf
+
+# To ensure reproducibility using Keras during development
+# https://keras.io/getting-started/faq/#how-can-i-obtain-reproducible-results-using-keras-during-development
+import numpy as np
+import random as rn
+
+# The below is necessary in Python 3.2.3 onwards to
+# have reproducible behavior for certain hash-based operations.
+# See these references for further details:
+# https://docs.python.org/3.4/using/cmdline.html#envvar-PYTHONHASHSEED
+# https://github.com/keras-team/keras/issues/2280#issuecomment-306959926
+randomState = 123
```

Comment:
This code was scraped from the blog that is referenced in order to try to get reproducible VAE models between training runs.  I am not exactly sure what each part of this is doing so that probably explains the redundancy.  If you think you have a better idea, I'm happy to sit down with you to do over it.